frontend/src/pages/vault/withdraw/Withdraw.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import './withdraw.css';
+import { ReactComponent as DiamondIcon } from 'assets/icons/diamond.svg';
+import { ReactComponent as TimeIcon } from 'assets/icons/time.svg';
+import { ReactComponent as SettingIcon } from 'assets/icons/settings.svg';
+
+export default function Withdraw() {
+  return (
+    <div className="withdraw-wrapper">
+      <div className="withdraw-container">
+        <div className="main-container">
+          <div className="top-cards">
+            <div className="withdraw-card">
+              <div className="card-header">
+                <span className="label">Total Amount staked</span>
+              </div>
+              <div className="card-value">
+                <span className="top-card-value">324,909,894</span>
+              </div>
+            </div>
```

Comment:
make it a separate component and reuse it 2 times in here, passing corresponding props

Reply:
*you should put it into components/ui folder and name it just `Card` so it can became a generic component that can be used in other places across the app